src/core/Akka.Persistence/persistence.conf

```diff
@@ -115,6 +115,9 @@ akka.persistence {
         # Dispatcher for message replay.
         replay-dispatcher = "akka.persistence.dispatchers.default-replay-dispatcher"
 
+        # Default serializer used as manifest serializer when applicable and payload serializer when no specific binding overrides are specified
+        serializer = "json"
+
```

Comment:
Here i defined the default serializer setting in the `journal-plugin-fallback`. Which should be used by default as the fallback plugin for every journal plugin, by the persistence subsystem. But I still need to verify this is actually true